Shake/Main.lean

```diff
@@ -454,3 +457,7 @@ def main (args : List String) : IO Unit := do
       out := out ++ text.extract insertion text.endPos
 
       IO.FS.writeFile path out
+
+  -- Since we throw an error upon encountering issues, we can be sure that everything worked
+  -- if we reach this point of the script.
+  println!"All suggestions applied successfully."
```

Comment:
Probably there should be a special case here in case the edit list is empty, e.g. `No edits required.` instead

Reply:
@Vierkantor, when you have a moment, do you want to make this change and then merge?

Reply:
Since there is the possibility that the edit list is nonempty but nothing gets applied, I ended up counting the number of edits in the edit-apply-loop.